### PR TITLE
add explicit ssh notes about users

### PR DESCRIPTION
**Pull request by EFM-Bobby**
This is a fix for https://enterprisedb.atlassian.net/browse/EFM-1459

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Comment by EFM-Bobby** [comment]
@Jagdish-Kewat-EDB Does this look to you like it fixes https://enterprisedb.atlassian.net/browse/EFM-1459 ?

File: product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx [hunk 1]
```diff
@@ -157,6 +157,7 @@ by root and that user/group/other (0755) has read and execute access. The script
 For the PgBouncer integration, passwordless `ssh` access is required. There are multiple ways 
 to configure `ssh`. Follow your organization's recommended process to 
 configure the passwordless `ssh`. For a quick start, you can also follow this example for configuring passwordless `ssh`.
+The 'efm' user will need to be able to ssh as the user running pgbouncer, i.e. the 'enterprisedb' user.
```
Comment (drothery-edb) [inline product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx:160]:
```suggestion
The user efm user must be able to ssh as the user running PgBouncer, enterprisedb.
```

File: product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx [hunk 2]
```diff
@@ -65,7 +65,7 @@ to those instructions:
 
 1. Create an integration script that connects to every (remote)
 PgBouncer host and runs the redirect script. Locate the script at `/usr/edb/efm-4.2/bin/efm_pgbouncer_functions`. Make sure the user
-efm can execute the script, which has the following contents:
+efm can execute the script, which has the following contents (note that the 'efm' user is ssh'ing as 'enterprisedb' to run the script):
```
[the remark this answers is not in the document]
Comment (EFM-Bobby) [reply product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx:68]: The "must" part doesn't make as much sense to me. This sentence is describing what's happening in the script, not what someone should do if writing their own or something. I just want to call attention to that detail in the script that follows.

File: product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx (hunk 1 above)
Comment (EFM-Bobby) [reply product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx:160]: Conceivably they might install pgbouncer as another user besides 'enterprisedb.' If we say elsewhere that PgBouncer has to be installed/run as enterprisedb then this is ok with me. Definitely need to fix the capitalization for PgBouncer though.

Comment (drothery-edb) [reply product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx]: How about: The user efm user must be able to ssh as the user running PgBouncer; for example, enterprisedb.

Comment (EFM-Bobby) [reply product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx]: Great!

File: product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (drothery-edb) [reply product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx:68]: How about:  The user efm runs ssh as enterpriseDB to run the script.

**Pull request merged**